### PR TITLE
Fix: Sony Channels are Viasat now

### PR DESCRIPTION
**Pull request by smrtprjcts**
Hi @freearhey,
Maybe  xmltv_ids should be modified, I don't know.
Thanks

Btw:
About channel name changes  https://hu.wikipedia.org/wiki/Viasat_2 https://hu.wikipedia.org/wiki/Viasat_Film

### CODE REVIEW

**Review by freearhey — CHANGES_REQUESTED** [review]
For everything to work correctly it is necessary to update not only the channel name and `xmltv_id` but also the `site_id` if it has changed.

File: sites/musor.tv/musor.tv_hu.channels.xml [hunk 1]
```diff
@@ -130,8 +130,8 @@
     <channel lang="hu" xmltv_id="RTLPlus.hu" site_id="RTL_PLUSZ">RTL+</channel>
     <channel lang="hu" xmltv_id="SkyNewsInternational.uk" site_id="SKYNEWS">Sky News</channel>
     <channel lang="hu" xmltv_id="SlagerTV.hu" site_id="SLAGERTV">Sláger TV (HD)</channel>
-    <channel lang="hu" xmltv_id="SonyMaxHungary.hu" site_id="SONY_MAX">Sony Max</channel>
-    <channel lang="hu" xmltv_id="SonyMovieChannelHungary.hu" site_id="SONY_MOVIE_CHANNEL">Sony Movie Channel</channel>
+    <channel lang="hu" xmltv_id="SonyMaxHungary.hu" site_id="SONY_MAX">Viasat2</channel>
```
Comment (freearhey) [inline sites/musor.tv/musor.tv_hu.channels.xml:133]:
```suggestion
    <channel lang="hu" xmltv_id="Viasat2.hu" site_id="VIASAT2">Viasat2</channel>
```

File: sites/musor.tv/musor.tv_hu.channels.xml [hunk 2]
```diff
@@ -130,8 +130,8 @@
     <channel lang="hu" xmltv_id="RTLPlus.hu" site_id="RTL_PLUSZ">RTL+</channel>
     <channel lang="hu" xmltv_id="SkyNewsInternational.uk" site_id="SKYNEWS">Sky News</channel>
     <channel lang="hu" xmltv_id="SlagerTV.hu" site_id="SLAGERTV">Sláger TV (HD)</channel>
-    <channel lang="hu" xmltv_id="SonyMaxHungary.hu" site_id="SONY_MAX">Sony Max</channel>
-    <channel lang="hu" xmltv_id="SonyMovieChannelHungary.hu" site_id="SONY_MOVIE_CHANNEL">Sony Movie Channel</channel>
+    <channel lang="hu" xmltv_id="SonyMaxHungary.hu" site_id="SONY_MAX">Viasat2</channel>
+    <channel lang="hu" xmltv_id="SonyMovieChannelHungary.hu" site_id="SONY_MOVIE_CHANNEL">Viasat Film</channel>
```
Comment (freearhey) [inline sites/musor.tv/musor.tv_hu.channels.xml:134]:
```suggestion
    <channel lang="hu" xmltv_id="ViasatFilm.hu" site_id="VIASAT_FILM">Viasat Film</channel>
```

**Comment by smrtprjcts** [comment]
Fix them

**Review by freearhey — APPROVED** [review]

**Pull request merged**